daemon/main.go

```diff
@@ -83,6 +86,61 @@ func main() {
 	}
 }
 
+func checkMinRequirements() {
+	log.Infof("Checking minimal requirements...")
+	clangVersion, err := exec.Command("clang", "--version").CombinedOutput()
+	if err != nil {
+		log.Fatalf("clang 3.8.x version: NOT OK: %s", err)
+	}
+	if !strings.Contains(string(clangVersion), "3.8") {
+		log.Fatalf("clang 3.8.x version: NOT OK, please install clang version 3.8.x in your system")
+	}
+	log.Infof("clang 3.8.x version: OK!")
+	lccVersion, err := exec.Command("llc", "--version").CombinedOutput()
+	if err == nil {
+		if strings.Contains(strings.ToLower(string(lccVersion)), "debug") {
+			log.Warningf("llc version was compiled in debug mode, expect higher latency!")
+		}
+	}
+	// /usr/include/gnu/stubs-32.h is installed by 'glibc-devel.i686' in fedora
+	// /usr/include/sys/cdefs.h is installed by 'libc6-dev-i386' in ubuntu
+	// both files exist on both systems but cdefs.h already exists in fedora
+	// without 'glibc-devel.i686' so we check for 'stubs-32.h first.
+	if _, err := os.Stat("/usr/include/gnu/stubs-32.h"); os.IsNotExist(err) {
```

Comment:
What's the best way to do it? I couldn't find the default include paths used by clang. I wouldn't like to do it like `find /usr/include -name stubs-32.h`.

Without `glibc-devel` I get this:

```
In file included from /var/run/cilium/bpf/bpf_netdev.c:27:
In file included from /var/run/cilium/bpf/include/bpf/api.h:11:
In file included from /var/run/cilium/bpf/include/linux/type_mapper.h:4:
In file included from /usr/bin/../lib64/clang/3.8.0/include/stdint.h:63:
In file included from /usr/include/stdint.h:25:
In file included from /usr/include/features.h:392:
/usr/include/gnu/stubs.h:7:11: fatal error: 'gnu/stubs-32.h' file not found
# include <gnu/stubs-32.h
```